samples/sensor/ccs811/README.rst

```diff
@@ -0,0 +1,81 @@
+.. _ccs811:
+
+CCS811 Indoor Air Quality Sensor
+################################
+
+Overview
+********
+
+The CCS811 digital gas sensor detects volatile organic compounds for
+indoor air quality measurements.  It estimates carbon dioxide where the
+main source of VOCs is human presence.
+
+Building and Running
+********************
+
+Building and Running on nrf52_pca20020
+======================================
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/sensor/ccs811
+   :board: nrf52_pca20020
+   :goals: build flash
+   :compact:
+
+
+Sample Output
+=============
+
+The sample below shows output from a Nordic Thingy:52 (nrf52_pca20020).
```

Comment:
How about instead:
```
The sample output below is from a `Nordic Thingy:52
<https://infocenter.nordicsemi.com/pdf/Thingy_PB_v1.0.pdf>`_
(nrf52_pca20020) that includes this sensor (and others).
After a soft reset, there is a 5-second startup period
where readings are unstable, and then we can see steady
reported measurements of about 400 ppm eC02 and 0 ppb eTVOC.
```

Reply:
Done, though I used a link to the product web page rather than to a PDF that might have to be downloaded in some browsers.